apps/web/src/pages/releases/index.tsx

```diff
@@ -73,13 +60,13 @@ export default function Releases({ packs }: PublishedPacks) {
   // If state changes, update the URL
   useEffect(() => {
     const previousState = stringify(parse(location.search))
-    const nextState = stringify(state)
+    const nextState = stringify(state as any)
```

Comment:
Sorry for this. I was getting this BS and as far as I can tell it's just a failing of Typescript casting.
```
Argument of type 'PackFilterState' is not assignable to parameter of type 'StringifiableRecord'.
  Index signature for type 'string' is missing in type 'PackFilterState'.
```

Reply:
What version of query-string are you running? I experienced this before bumping it to `7.1.1` in a client project. If you don't have any luck, you can also do `state as unknown as PackFilterState`.

Reply:
Ah! Apparently I hadn't run `npm i` That did the trick.